src/main.cpp

```diff
@@ -241,14 +235,17 @@ Napi::Value DDWAFContext::run(const Napi::CallbackInfo& info) {
     mlog("Set total_runtime");
     res.Set("totalRuntime", Napi::Number::New(env, result.total_runtime));
   }
-  if (code != DDWAF_GOOD) {
+  if (code != DDWAF_OK) {
     res.Set("data", Napi::String::New(env, result.data));
   }
-  if (code == DDWAF_MONITOR) {
-    res.Set("action", Napi::String::New(env, "monitor"));
-  }
-  if (code == DDWAF_BLOCK) {
-    res.Set("action", Napi::String::New(env, "block"));
+  if (code == DDWAF_MATCH) {
```

Comment:
These two if statements can be merged as `!= DDWAF_OK` is either `DDWAF_MATCH` or an error, but you already handled those above.